docs.html

```diff
@@ -373,7 +373,165 @@ <h2 class="list-topic">Components</h2>
 </code>
 </section>
 
-      </div>
+<section class="section" >
+  <div class="component-title" >Card</div>
+  <div class="component-detail" >
+
+    Cards are surfaces that display content and actions on a single topic.
+    They should be easy to scan for relevant and actionable information. Elements, like text and images, should be placed on them in a way that clearly indicates hierarchy.
+  </div>
+
+<div class="component-subhead" >Basic Card</div>
+
+
+<div class="card-container">
+  <img
+    src="https://picsum.photos/380/240"
+    alt="nature img"
+    class="random-img"
+    loading="eager"
+  />
+  <div class="card-details">
+    <h3 class="card-title">Our Chaning Work</h3>
+    <p class="card-subtitle">by tanishk</p>
+    <p class="card-text">
+      Visit ten places on our planet that are is smart undergoing the
+      biggest changes today.
+    </p>
+  </div>
+  <div class="card-options">
+    <button class="card-btn">READ</button>
+    <button class="card-btn">BOOKMARK</button>
+    <i class="far fa-heart card-icon"></i>
+    <i class="fas fa-share-alt card-icon"></i>
+    <i class="fas fa-ellipsis-v card-icon"></i>
+  </div>
+</div>
+
+
+
```

Comment:
remove unnecessary white lines from code, using some code formatter will solve the problem

Reply:
okay I will find a code formatter to solve this